src/main/java/com/shevtsov/services/ClientService.java

```diff
@@ -1,12 +1,48 @@
 package com.shevtsov.services;
 
 public interface ClientService {
+
+    /**
+     * Creates and transmits a Client-object to DAO for saving.
+     * @param name the name of new client.
+     * @param surname the surname of new client.
+     * @param phone the phone number of new client.
+     * @return true if new client was saved successfully, or false otherwise.
+     */
+    boolean createClient(String name, String surname, String phone);
+
     /**
-     * !!! написать к каждому методу описание
```

Comment:
ru detected

Reply:
it`s just remarks for myself, I will try to avoid it in future